core/build.gradle

```diff
@@ -9,14 +9,14 @@ apply plugin: 'jacoco'
 
 dependencies {
     compileOnly "org.opensearch:opensearch:${opensearch_version}"
-    compile "org.jetbrains.kotlin:kotlin-stdlib:${kotlin_version}"
-    compile 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.1.1'
-    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:${kotlin_version}"
-    compile "com.cronutils:cron-utils:9.1.6"
-    compile "org.opensearch.client:opensearch-rest-client:${opensearch_version}"
-    compile 'com.google.googlejavaformat:google-java-format:1.10.0'
-    compile "org.opensearch:common-utils:${common_utils_version}"
-    compile 'commons-validator:commons-validator:1.7'
+    api "org.jetbrains.kotlin:kotlin-stdlib:${kotlin_version}"
```

Comment:
why `api` and not `implementation` configuration? Are we exporting all of these libraries?

Reply:
At first I was trying to get the build working with gradle 7, but didnt realize that it would work if some of these can be converted to `implementation`. I converted as many of the dependencies from `api` to `implementation`

Reply:
I think alerting should just build fine. Now it will not export these dependencies transitively and thus will not be available to any other module, dependent on alerting, at compile time. But looking at these dependencies, I think that most of them, if not all, will not be required by other modules as transitive dependencies at compile time.